src/Weasel.Postgresql.Tests/Tables/detecting_table_deltas.cs

```diff
@@ -205,6 +205,23 @@ public async Task detect_extra_index()
             await AssertNoDeltasAfterPatching();
         }
 
+        [Fact]
+        public async Task ignore_new_index_that_is_ignored()
+        {
+            var existing = new detecting_table_deltas().theTable;
+            existing.ModifyColumn("user_name").AddIndex(idx => idx.Name = "ignore_me");
+
+            await CreateSchemaObjectInDatabase(existing);
```

Comment:
Move this line of code *before* you call `ModifyColumn()` up above and add the index. Don't think this test does anything otherwise

Reply:
I'm not sure which line shall be moved up.  I hope the test works as intended:  If I reverse the change in `TableDelta.cs`, it fails.

![grafik](https://user-images.githubusercontent.com/1508058/169563822-c52a117c-1d24-40b6-be03-f87688f0205c.png)

Note that I'm using two `Table` variables here:  `existing` is used to create the table *with* the index, `theTable` is then used to generate the delta while ignoring that (existing) index.